### ISSUE TITLE
Creating N enrollments for the same mission 

### ISSUE THREAD

**Issue by rodrigopinto** [playthecall/playthecall-2013#7]
I was creating an enrollment to a mission with my user when I realized that is was duplicated on database.

``` shell
User.first.mission_enrollments.map { |me| [me.id,  me.mission_id, me.url] }
  User Load (1.9ms)  SELECT "users".* FROM "users" LIMIT 1
  MissionEnrollment Load (0.6ms)  SELECT "mission_enrollments".* FROM "mission_enrollments" WHERE "mission_enrollments"."user_id" = 1
  Mission Load (0.5ms)  SELECT "missions".* FROM "missions" WHERE "missions"."id" = 21 LIMIT 1
  Mission Load (0.7ms)  SELECT "missions".* FROM "missions" WHERE "missions"."id" = 21 LIMIT 1
  Mission Load (0.5ms)  SELECT "missions".* FROM "missions" WHERE "missions"."id" = 21 LIMIT 1
  Mission Load (0.7ms)  SELECT "missions".* FROM "missions" WHERE "missions"."id" = 21 LIMIT 1
 => [[4, 21, "m/rodrigoospinto/mission-1"], [3, 21, "m/rodrigoospinto/mission-1"], [2, 21, "m/rodrigoospinto/mission-1"], [1, 21, "m/rodrigoospinto/mission-1"]]
```

I am not sure if it is part of the business rule or not, I'd like to make a fix for that, but I wasn't sure about that.

**Comment by badosu** [comment]
No @rodrigopinto, it is indeed a bug.

How did you enroll in the mission?

**Comment by eniolopes** [comment]
Fixed by 31974b21456634772ed485bbd065d2e927cebd5e

**Comment by badosu** [comment]
Awesome!
